6_visualize/src/create_gif_tasks.R

```diff
@@ -52,6 +63,7 @@ create_timestep_gif_tasks <- function(timestep_ind, folders){
         "legend_fun,",
         "watermark_fun,",
         "gage_sites_plot_fun_%s,"=cur_task$tn,
+        "callouts_plot_fun_%s,"=cur_task$tn,
```

Comment:
It would be a little more in keeping with the local convention if gage_sites_plot_fun_xx and callouts_plot_fun_xx were named gage_sites_fun_xx and callouts_fun_xx, respectively; the `plot` part is established by context.